### ISSUE TITLE
anchorY="bottom-baseline" not working since v0.48.0

### ISSUE THREAD

**Issue by ffdead** [protectwise/troika#309]
Seems like something changed between v0.47.2 and v0.48.0 related to how the bottom-baseline is calculated.

I noticed this when updating from @react-three/drei@9.97.0 which has (troika-three-text@0.47.2 as a dependency) to @react-three/drei@9.105.2 which has troika-three-text@0.49.0 as a dependency. 

By manually overriding the troika dependincy using the resolutions field in package.json is was able to track down that this happened in v0.48.0

```jsx
  <Text
        fontSize={scale}
        sdfGlyphSize={128}
        font={font.path}
        anchorY="bottom-baseline" 
        lineHeight={1}
  />
  ```

v0.47.2
<img width="657" alt="Screenshot 2024-04-09 at 16 32 08" src="https://github.com/protectwise/troika/assets/420472/7312093e-b337-45c3-9c01-2d892b4a7c46">

v0.48.0
<img width="610" alt="Screenshot 2024-04-09 at 16 32 53" src="https://github.com/protectwise/troika/assets/420472/2dc2ab7b-510a-42b7-8e58-22f7c35ae900">

**Comment by lojjic** [comment]
Thanks! I'd missed a negative sign. 🤦‍♂️ I've published 0.49.1 with the fix.

**Comment by ffdead** [comment]
Amazing, thank you @lojjic - I can confirm it works 🎉
